definitions/scenarios/backup.rb

```diff
@@ -69,6 +69,8 @@ def set_context_mapping
                   Procedures::Backup::Metadata => :incremental_dir)
       context.map(:proxy_features,
                   Procedures::Backup::ConfigFiles => :proxy_features)
+      context.map(:strategy,
```

Comment:
The context map should be replaced by passing the :ignore_changed_files param in compose e.g.
```ruby
add_step_with_context(Procedures::Backup::ConfigFiles, :ignore_changed_files => true)
```
in `add_online_backup_steps`.